index.html

```diff
@@ -6025,7 +6025,7 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
           <li>None of the <rref>option</rref> elements in the <rref>listbox</rref> have an explicitly declared value for <sref>aria-selected</sref> or <sref>aria-checked</sref>.</li>
         </ul>
         <p>
-          If a user agent provides an implicit <sref>aria-selected</sref> value for an <rref>option</rref>, the value SHOULD be <code>true</code> if the <rref>option</rref> has DOM focus or the <rref>listbox</rref> has DOM focus and the <rref>option</rref> is referenced by <pref>aria-activedescendant</pref>.
+          If a user agent provides an implicit <sref>aria-selected</sref> value for an <rref>option</rref>, the value SHOULD be <code>true</code> if the <rref>option</rref> has DOM focus or has <code>tabindex="0"</code> (or <code>tabindex</code> with a value greater than 0), or the <rref>option</rref> is referenced by <pref>aria-activedescendant</pref>.
```

Comment:
trying to simplify this (DOM focus, tabindex=0, tabindex=1+ all seem to just imply the option has keyboard focus... am i missing something to think otherwise?) and bringing in James's requested change.

```suggestion
          If a user agent provides an implicit <sref>aria-selected</sref> value for an <rref>option</rref>, the value SHOULD be <code>true</code> if the <code>option</code> has keyboard focus, or the <code>option</code> is referenced by <pref>aria-activedescendant</pref> so long as the <code>option</code> is not hidden or inert.
```

Reply:
@scottaohara But this would not solve the problem of #1661, that a listbox has a selected option only if it has keyboard focus (unless it uses aria-selected or aria-activedescandant). #1661 can only be solved with additional conditions for the non-focused state or by removing implicit aria-selected.

Reply:
If we define hidden and inert as exclusion reasons, then disabled should be one as well

Reply:
i'm resolving this as talkign with Sarah about this, we understand why this won't work.